docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -164,73 +247,151 @@ Using the detailed script below as a simulator, there are three behaviors:
         ```   
         {"cmd":"randnum", "method":"get", "uuid":"293d7a00-66e1-4374-ace0-07520103c95f", "randnum":42.0}
         ```
-3.  Receive the set request, then change the device value.
 
+    **Using Multi-level Topic:**
+    
+    1. The simulator receives the request from the MQTT broker, the topic is `command/my-custom-device/randnum/get/293d7a00-66e1-4374-ace0-07520103c95f` and message is similar to the following:
+    
+        ```
+        {"randnum":"42.0"}
+        ```
+    
+    2. The simulator returns the response to the MQTT broker, the topic is `command/response/#` and the message is similar to the following:
+    
+        ```
+        {"randnum":"4.20e+01"}
+        ```
+    
+3. Receive the set request, then change the device value.
+
+    **Default (single-level) Topic:**
+    
     1. The simulator receives the request from the MQTT broker, the topic is `CommandTopic` and the message is similar to the following:
+    
         ```   
         {"cmd":"message", "method":"set", "uuid":"293d7a00-66e1-4374-ace0-07520103c95f", "message":"test message..."}
         ```
+    
     2. The simulator changes the device value and returns the response to the MQTT broker, the topic is `ResponseTopic` and the message is similar to the following:
+    
         ```   
         {"cmd":"message", "method":"set", "uuid":"293d7a00-66e1-4374-ace0-07520103c95f"}
         ```
+    
+    **Using Multi-level Topic:**
+    
+    1. The simulator receives the request from the MQTT broker, the topic is `command/my-custom-device/testmessage` and the message is similar to the following:
```

Comment:
the same topic change should also be applied to the set command

Reply:
yes, done.